documentation/src/main/asciidoc/elasticsearch-integration.asciidoc

```diff
@@ -122,6 +116,24 @@ Let's see the options for the `index_schema_management_strategy` property:
 |`RECREATE`|Indexed will be deleted if existing and then created. This will delete all content from the index!
 |`RECREATE_DELETE`|Similarly to `RECREATE` but will also delete the index at shutdown. Commonly used for tests.
 |===============
+Maximum time to wait for the indexes to become available before failing (in ms):: `hibernate.search.default.elasticsearch.index_management_wait_timeout 10000` (default)
+Status an index must at least have in order for Hibernate Search to work with it (one of "green", "yellow" or "red")::
+`hibernate.search.default.elasticsearch.required_index_status green` (default)
++
+Only operate if the index is at this level or safer.
+In development, set this value to `yellow` if the number of nodes started is below the number of expected replicas.
```

Comment:
Why would you suggest this? Seems quite a bad practice, you have the option to start your ES node as a single node, so it will be green, and faster.